src/mongocxx/collection.hpp

```diff
@@ -119,10 +118,8 @@ class MONGOCXX_API collection {
     ///
     /// @see http://docs.mongodb.org/manual/reference/command/aggregate/
     ///
-    cursor aggregate(
-        const pipeline& pipeline,
-        const options::aggregate& options = options::aggregate()
-    );
+    cursor aggregate(const pipeline& pipeline,
```

Comment:
Attack of the clang format.  Is clang format something we want to apply to this codebase, or not?  If not I will fix the style changes here.